lib/optimizer/tests/Transformer/ServerSideRenderingTest.php

```diff
@@ -265,7 +265,7 @@ public function dataTransform()
             'bad heights attribute' => [
                 $input('<amp-img height="256" heights=",,," layout="responsive" width="320"></amp-img>'),
                 // This adds an ID as it stores the CSS to inline before the actual error is detected.
-                $expectWithBoilerplate('<amp-img height="256" heights=",,," layout="responsive" width="320" id="i-amp-0"><i-amphtml-sizer style="display:block;padding-top:75.0000%;"></i-amphtml-sizer></amp-img>'),
+                $expectWithBoilerplate('<amp-img class="i-amphtml-layout-responsive i-amphtml-layout-size-defined" height="256" heights=",,," i-amphtml-layout="responsive" layout="responsive" width="320"><i-amphtml-sizer style="display:block;padding-top:80.0000%;"></i-amphtml-sizer></amp-img>'),
```

Comment:
Difference is `id` is removed, while these are added:

* `class="i-amphtml-layout-responsive i-amphtml-layout-size-defined"`
* `i-amphtml-layout="responsive"`

And the `padding-top` changes from 75% to 80%.

Reply:
`256 / 320 = 0.80`

Reply:
Matches output with Node.js Optimizer.